webapp/app/templates/lotse/form_familienstand.html

```diff
@@ -4,7 +4,7 @@
 {% block form_content %}
 
     <div id="familienstand_field" class="mb-3">
-        {{ macros.render_field(form.familienstand, 10) }}
+        {{ macros.render_field(form.familienstand, 10, first_field=True) }}
```

Comment:
I don't think the autofocus is working as intended? At least, it's not focused for me locally. Same with the checkboxes and the entries field.

....and I just now realized why you asked for the special fields.. 🤦

Reply:
So, we probably should talk about this?

Reply:
I'll look into this